Foundation/NSString.swift

```diff
@@ -457,7 +457,10 @@ extension NSString {
     
     public var intValue: Int32 {
         get {
-            NSUnimplemented()
+            let scanner = NSScanner(string: _swiftObject)
```

Comment:
with the experimental API currently implemented, this could just be one line: `return NSScanner(string: _swiftObject).scanInt() ?? 0`